set.mm

```diff
@@ -42842,14 +42919,23 @@ will do (e.g., ` O = (/) ` and ` T = { (/) } ` , see ~ 0nep0 ).
     ralsng.1 $e |- ( x = A -> ( ph <-> ps ) ) $.
     $( Substitution expressed in terms of quantification over a singleton.
        (Contributed by NM, 14-Dec-2005.)  (Revised by Mario Carneiro,
-       23-Apr-2015.)  (Proof shortened by AV, 7-Apr-2023.) $)
+       23-Apr-2015.)  (Proof shortened by AV, 7-Apr-2023.)  Avoid ~ ax-10 ,
```

Comment:
I think "(Proof shortened by AV, 7-Apr-2023.)" can be removed here. I do not know if this should become a general rule (if a proof is modified extensively, then former proof shortenings become obsolete), but in this case the proof became much longer...

Reply:
This also holds for the three following changes.

Reply:
> I think "(Proof shortened by AV, 7-Apr-2023.)" can be removed here. I do not know if this should become a general rule

Probably doesn't need to be a general rule (hard to define standard criteria to decide when it's appropriate), but I agree that it makes sense in this case, I added a new commit [2440bb4](https://github.com/metamath/set.mm/pull/4256/commits/2440bb40ab5ac27c8bfba9bf9fab77607680ba33)